web-app/js/portal/ui/MainPanel.js

```diff
@@ -46,15 +46,15 @@ Portal.ui.MainPanel = Ext.extend(Ext.Panel, {
 
         Portal.ui.MainPanel.superclass.constructor.call(this, config);
 
-        Ext.MsgBus.subscribe(PORTAL_EVENTS.VIEW_DATA_COLLECTION, this._onViewGeoNetworkRecord, this);
-        Ext.MsgBus.subscribe(PORTAL_EVENTS.DATA_COLLECTION_ADDED, this._onGeoNetworkRecordAdded, this);
+        Ext.MsgBus.subscribe(PORTAL_EVENTS.VIEW_DATA_COLLECTION, this._onViewMetadataRecord, this);
```

Comment:
I agree that it is different to normal events in that way - it's a command rather than an announcement. If people are happy I will have a look into it (as a separate PR) and see if it can be done in a better way.